### ISSUE TITLE
The loss does not decrease

### ISSUE THREAD

**Issue by wtongping** [wondervictor/WESPE.pytorch#6]
thanks for your work! This code is runnable, but the loss(both the content, tv, color, texture loss) does not decrease when I training the net, and my test results is strange. Could you give me some advice or share your pre-trained models and test code? thank u !

**Comment by divyam-goel** [comment]
@wtongping I am facing the same issue. Did you find a solution?
